service/handlers.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	// "github.com/pkg/errors"
+	"github.com/rancher/webhook-service/drivers"
+	"github.com/rancher/webhook-service/webhooks"
+	"io/ioutil"
+	"net/http"
+)
+
+type WebhookRequest map[string]interface{}
+
+func ConstructPayload(w http.ResponseWriter, r *http.Request) {
+	var webhookRequestData WebhookRequest
+	log.Infof("Construct Payload")
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Errorf("Construct failed with error: %v", err)
+		return
+	}
+	json.Unmarshal(bytes, &webhookRequestData)
+	jwt, err := webhooks.ConstructPayload(webhookRequestData)
+	if err != nil {
+		fmt.Fprintf(w, "%v", err)
+	}
+	fmt.Fprintf(w, "%v\n", jwt)
+}
+
+func Execute(w http.ResponseWriter, r *http.Request) {
+	var payload WebhookRequest
+	log.Infof("Execute")
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Errorf("Construct failed with error: %v", err)
+		return
+	}
+	json.Unmarshal(bytes, &payload)
+
+	// var webhookDriver webhooks.Driver
+	_, resp, err := webhooks.Execute(payload)
```

Comment:
You need to use the response code as the HTTP response code to return in the HTTP response to the user.